tests/unit/constants_test.py

```diff
@@ -0,0 +1,8 @@
+from yarpc import constants
+
+
+def test_NoValue_object():
+    """
+    Should ensure the NoValue property is an object
+    """
+    assert type(constants.NoValue) == object
```

Comment:
What's the point of this test? What could possibly go wrong? This variable just has to be unique. Also I'd prefer `assert isinstanse(NoValue, object)`

Reply:
The point of this test is that knowing that this value has to be unique we must check that a variable is always an object, which is what we want in that case.